text/3463-crates-io-policy-update.md

```diff
@@ -0,0 +1,224 @@
+current owner directly. If the current owner agrees, they can add you as an
+owner of the crate, and you can then remove them, if necessary. If the current
+owner is not reachable or has not published any contact information the
+crates.io team may reach out to help mediate the process of the ownership
+transfer.
+
+The crates.io team only deletes crates from the registry that do not comply with
+the policies on this document. Crate deletion by their owners is not possible to
+keep the registry as immutable as possible. If you want to flag your crate as
+open for transferring ownership to others, you can publish a new version with
+a message in the README or description communicating to the crates.io support
+team that you consent to transfer the crate to the first person who asks for it:
+
+> I consent to the transfer of this crate to the first person who asks
+> help@crates.io for it.
+
+
+## Data Access
+
+If you need access to a large subset of the crates.io database we recommend
+first looking at the **crates.io [index repository](https://github.com/rust-lang/crates.io-index)**.
+This repository is updated live whenever new versions are published and contains
+all the information needed for cargo to run the dependency resolution algorithm.
+
+In case the index dataset is insufficient for your purposes, we also publish a
+**database dump** every 24 hours. This includes the majority of data from our
+database except for sensitive private information. The latest database dump is
+available at <https://static.crates.io/db-dump.tar.gz> and information on using
+the content is contained in the tarball. Please note that while we aim to keep
+the data structure somewhat stable, we can not give any stability guarantees on
+the exact database table layouts.
+
+If the index repository and the database dump are insufficient you may also use
+the crates.io API directly, though it is at the discretion of the crates.io to
+block any excessive usage. We require users of the crates.io API to limit
+themselves to a maximum of 1 request per second.
+
+We also require all API users to provide a user-agent header that allows us to
+uniquely identify your application. This allows us to more accurately monitor
+any impact your application may have on our service. Providing a user agent that
+only identifies your HTTP client library (such as `request/0.9.1`) increases the
+likelihood that we will block your traffic.
+
+It is recommended, to include contact information in your user-agent header:
+
+- Bad: `User-Agent: reqwest/0.9.1`
+- Better: `User-Agent: my_bot`
+- Best: `User-Agent: my_bot (my_bot.com/info)` or `User-Agent: my_bot (help@my_bot.com)`
+
+This allows us to contact you if we would like a change in your application's
+behavior without having to block your traffic.
+
+We reserve the right to block traffic from any client that we determine to be in
+violation of this policy or causing an impact on the integrity of our service.
+
+
+## Security
```

Comment:
Worth adding a caveat to this section about responsible disclosure to crate authors, since this is only for cargo and crates.io themselves. Just something simple like, you should look at the documentation for individual crates on how individual authors prefer to disclose vulnerabilities.

Reply:
that sounds like a good idea to me. do you have a specific wording in mind already?

Reply:
I'm on my phone right now, so, I'll just add a quick paragraph and you can incorporate it:

> Note that this policy only applies to official Rust projects like crates.io and cargo, and not individual crates. The Rust Foundation is not responsible for the disclosure of vulnerabilities to specific crates, and if any issues are found, you should see guidance from the crate owners and their specific policies instead.